.github/workflows/build-images.yml

```diff
@@ -224,7 +224,7 @@ jobs:
           rm -rf "scripts/ci"
           mv "main-airflow/scripts/ci" "scripts"
       - name: "Free space"
-        run: ./scripts/ci/tools/free_space.sh
+        run: ./scripts/ci/tools/freespace.py
```

Comment:
Comment here: for all the "freespace.py" script, you will also have to install dependenciesl (those that are specified in setup.py and setup.cfg in dev/breeze

The setup-python action supports caching dependencies: 
* https://github.com/actions/setup-python#caching-packages-dependencies

So you'd have to add something like I've done here: https://github.com/apache/airflow/blob/4ac35d723b73d02875d56bf000aafd2235ef0f4a/.github/workflows/ci.yml#L240 for every "setup-python" step in the ci.yml.

Reply:
I got it: Now we can run:  `python3 freespace.py` taking the VERBOSE variable environment.  Thank you, Jarek.

Reply:
What you mean is that freespace scripts need python dependencies like click, rich, and others, so with setup-python@v2 I am installing these dependencies on the environment. So I added a "uses" in each step where we call freesapce. Because also I was having an issue like this:

`Traceback (most recent call last):
  File "./scripts/ci/tools/freespace.py", line 27, in <module>
    from rich.console import Console
ModuleNotFoundError: No module named 'rich'`

let's see how it will go now :)

Reply:
I think you also need to add `run: pip install  ./dev/breeze` as the next step.

Also I think we could to one more thing - the "free_space.py" script shoudl be moved to `./dev/breeze/ci` (and basically all other python scripts. This way we will keep them as part of the "Breeze" package, we will keep some common dependencies and it will be possible to reuse the code from breeze - for example what @Bowrna  is adding now is the util "run_command", and if you move "free_space.py" part of the breeze package, the common utils will be reusble.

Reply:
I understand that we want to separate Breeze from Airflow, correct? Breeze for the development environment, which will have its own ci scripts and documentations.